### ISSUE TITLE
WeMos D1 R2 - XTENSA-architecture microcontrollers are not supported

### ISSUE THREAD

**Issue by ezwyj** [TheAustrian/ESP8266-LED-Websockets#4]
#2

 tryed this and other options but got an error.

env
Board: WeMos D1 R2  
Arduino: v1.6.8
Arduino core for ESP8266: v2.3.0
FastLED: V3.1.0

all is new versions

ERROR: XTENSA-architecture microcontrollers are not supported.
exit status 1
Error compiling for board WeMos D1 R2 & mini.

In file included from D:\CODE_ARDUINO\libraries\FastLED-3.1.0/FastLED.h:34:0,

```
             from C:\Users\yuejunw\AppData\Local\Temp\arduino_modified_sketch_560590\FirstLight.ino:6:
```

D:\CODE_ARDUINO\libraries\FastLED-3.1.0/led_sysdefs.h:22:2: error: #error "XTENSA-architecture microcontrollers are not supported."

 #error "XTENSA-architecture microcontrollers are not supported."

  ^

In file included from D:\CODE_ARDUINO\libraries\FastLED-3.1.0/FastLED.h:42:0,

```
             from C:\Users\yuejunw\AppData\Local\Temp\arduino_modified_sketch_560590\FirstLight.ino:6:
```

D:\CODE_ARDUINO\libraries\FastLED-3.1.0/platforms.h:22:2: error: #error "XTENSA-architecture microcontrollers are not supported"

 #error "XTENSA-architecture microcontrollers are not supported"

  ^

exit status 1
Error compiling for board WeMos D1 R2 & mini.

**Comment by ezwyj** [comment]
In file included from D:\CODE_ARDUINO\libraries\FastLED-3.1.0/FastLED.h:34:0,

```
             from C:\Users\yuejunw\AppData\Local\Temp\arduino_modified_sketch_560590\FirstLight.ino:6:
```

D:\CODE_ARDUINO\libraries\FastLED-3.1.0/led_sysdefs.h:22:2: error: #error "XTENSA-architecture microcontrollers are not supported."

 #error "XTENSA-architecture microcontrollers are not supported."

  ^

In file included from D:\CODE_ARDUINO\libraries\FastLED-3.1.0/FastLED.h:42:0,

```
             from C:\Users\yuejunw\AppData\Local\Temp\arduino_modified_sketch_560590\FirstLight.ino:6:
```

D:\CODE_ARDUINO\libraries\FastLED-3.1.0/platforms.h:22:2: error: #error "XTENSA-architecture microcontrollers are not supported"

 #error "XTENSA-architecture microcontrollers are not supported"

  ^

exit status 1
Error compiling for board WeMos D1 R2 & mini.

**Comment by TheAustrian** [comment]
You closed the issue, so I guess you found the solution as described here: https://github.com/FastLED/FastLED/issues/318#issuecomment-227046181

Cheers, Al
